File: AddevenPosition.java
import java.util.*;

public class AddevenPosition {
    public static void main(String args[]){
    Scanner sc = new Scanner (System.in);
    int n= sc.nextInt();
 int evensum=0 ;int oddsum=0;
    
    //System.out.println(count);
    for (int i = 0; i < String.valueOf(n).length(); i++) {
        int digit = Character.getNumericValue(String.valueOf(n).charAt(i));
        if (i % 2 == 0) {
            evensum += digit;
        } else {
            oddsum += digit;
        }}
        System.out.println("Sum of even position numbers: " + evensum);
        System.out.println("Sum of odd position numbers: " + oddsum);
    
}}

File: addevenposition.cpp

#include <iostream>

using namespace std;

int main() {
    int n = 2345678;
    int evenSum = 0;
    int oddSum = 0;
    
    for (int i = 0; i < to_string(n).length(); i++) {
        
        int digitValue = (int)(to_string(n)[i] - '0');
        if (i % 2 == 0) {
            evenSum += digitValue;
        } else {
            oddSum += digitValue;
        }
    }
    
    cout << "Sum of even-positioned digits: " << evenSum << endl;
    cout << "Sum of odd-positioned digits: " << oddSum << endl;
    
    return 0;
}



